test/fixtures/agenda-instance.js

```diff
@@ -1,28 +1,30 @@
-var connStr = process.argv[2];
-var tests = process.argv.slice(3);
-
-var path = require('path'),
-    Agenda = require( path.join(__dirname, '..', '..', 'index.js' ) ),
-    addTests = require( path.join(__dirname, 'addTests.js') );
-
-var agenda = new Agenda({ db: { address: connStr } }, function(err, collection) {
-
-	tests.forEach(function(test) {
+const Agenda = require('../../index');
```

Comment:
The reason I switched to this is if we want to move to ES6 with babel in the future to allow for tree shaking we would likely end up using `import x from '../x';` and you can't use dynamic file paths with imports. Also it's for the most part considered a bad idea to use dynamic requires like that.

Maybe we should open an issue for this discussion so we can add an eslint plugin to force the chosen way?